specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/definitions.json

```diff
@@ -9232,13 +9232,60 @@
         "description": {
           "type": "string",
           "description": "Description of the workspace."
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of workspace.",
+          "enum": [
+            "KubernetesManaged",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "GatewayKind",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "KubernetesManaged",
+                "description": "Configuration of traffic routes in workspace is managed by Kubernetes resources."
+              },
+              {
+                "value": "Default",
+                "description": "Configuration of APIs in workspace is managed through Azure Resource Manager (ARM)."
+              }
```

Comment:
This is a massive nit, but let's put the default first so that it's the first item in generated enums as well (and sorted alphabetically)

Reply:
That's fair.